add v3 staking panel style

ayumitk opened the pull request:
**Pull Request Summary**

Added styles for the staking panel on the assets page

<img width="978" alt="Screenshot 2024-01-14 at 12 57 45 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/83b22b4d-f288-4e98-b945-7283b878ff89">

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**Remaining Todos:**
- [ ] Add a tooltip to the remaining Days (@ayumitk will add it in another PR)

**Need logics:**
- [x] Period
- [x] Remaining days
- [x] Available to withdraw amount
- [x] Re-lock amount

<img width="939" alt="Screenshot 2024-01-14 at 1 16 12 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/fa0c6c40-136b-4d2b-a0be-7484cbab65a9">

<img width="922" alt="Screenshot 2024-01-14 at 1 15 36 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/9a122d50-7c4d-456d-b9e1-5c1df397296c">

Kahonnohak commented:
All looks good to me!

One small thing on mobile to adjust if you have time
1) Available to Vote -> 2 lines maybe?
2) Unbonding amount -> bottom fix
3) Remaining days -> Centre fix

And sorry my mistake
4) no days needed for the bonus (My Staking)
5) Are you adding this in this PR?


@bobo-k2 
Could you please help adding logics for 

My Staking
1) Add period
2) Add eras on the days (its technically different but please let us use days in the UI)

My dApps
 3) Would it be possible to separate the staking per projects also if the amount has bonus or not?
 eg. 
 dApp A  xxxx ASTR  Bonus
 dApp A  xxxx ASTR      -    

Unbonding
4) Available to withdraw
5) Re-lock + staking modal

bobo-k2 commented:
> My dApps
> 3) Would it be possible to separate the staking per projects also if the amount has bonus or not?
eg.
> dApp A xxxx ASTR Bonus
> dApp A xxxx ASTR -

I am sorry, I don't get the above

ayumitk commented:
@Kahonnohak I've fixed 1,2,3,4.

> 5) Are you adding this in this PR?

sorry, I don't get this. What does this mean?

ayumitk commented:
> At certain browser widths (˜1300px, ˜1050px, ˜700px), arrows arrangement looks weird

@bobo-k2 I've fixed this. And thank you for adding logics🙏

Kahonnohak commented:
> @Kahonnohak I've fixed 1,2,3,4.
> 
> > 5. Are you adding this in this PR?
> 
> sorry, I don't get this. What does this mean?

oops sorry did not notice the message, its the tooltip. It is okay not to include in this PR.

<img width="916" alt="Screenshot 2024-01-15 at 16 51 26" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/431c0cd7-4ef2-4f5d-a8b2-97698c3d466e">

Kahonnohak commented:
@ayumitk thank you for the fix! I am so sorry, i missed a few places. 

1) Dark theme has not been configured
<img width="388" alt="Screenshot 2024-01-16 at 16 02 36" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/121ebe54-564c-4034-88c8-f11ba7d2ff5f">


2) No relation to this PR but the banner is covering the account area
<img width="396" alt="Screenshot 2024-01-16 at 16 02 06" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/cbba4880-5410-49fb-9b23-c7cb52299a4f">


3) some spacing - sorry
 A) Just realise why I did not do this, it is much better to have all balances at the right side, so please change the position of Remaining era <-> amount, also Bonus <-> amount on Unbonding and my dApp section
 
 B) On mobile, the SBY box SBY has a big space on the right but this is because of ASTR position? Would like all amount lined and but ticker has 4 or 3 letter.
 
 C) please make here 32px same as other box
 
 4) Tab underline is too long on mobile, if you could easily adjust please do
<img width="394" alt="Screenshot 2024-01-16 at 16 50 43" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/419b66e5-3ddd-4034-977f-bbc291c546a1">
<img width="387" alt="Screenshot 2024-01-16 at 16 02 17" src="https://github.com/AstarNetwork/astar-apps/assets/77480847/c0a05bf4-d4bc-4067-9308-dffd77b6dbfe">

ayumitk commented:
> A) Just realise why I did not do this, it is much better to have all balances at the right side, so please change the position of Remaining era <-> amount, also Bonus <-> amount on Unbonding and my dApp section

@Kahonnohak If I switch the Remaining era <-> amount position, how about the Available to withdraw and Re-lock section? It would be like this: 

<img width="895" alt="Screenshot 2024-01-16 at 1 11 39 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/a5b0a7ce-bdb3-417f-8d59-fa471a4504b6">

ayumitk commented:
> B) On mobile, the SBY box SBY has a big space on the right but this is because of ASTR position? Would like all amount lined and but ticker has 4 or 3 letter.

@Kahonnohak Yes, it's because of ASTR. Also, the amounts and symbols are vertically lined. Here is the current layout.

<img width="375" alt="Screenshot 2024-01-16 at 1 32 03 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/373c4224-3fb9-4892-9021-acd1294fdffe">

We can make amounts and symbols align right if you are okay with not vertically lined like this.

<img width="377" alt="Screenshot 2024-01-16 at 1 42 34 PM" src="https://github.com/AstarNetwork/astar-apps/assets/33358068/a2532352-3798-48b7-88d4-a0254ca97b39">

Kahonnohak commented:
> > A) Just realise why I did not do this, it is much better to have all balances at the right side, so please change the position of Remaining era <-> amount, also Bonus <-> amount on Unbonding and my dApp section
> 
> @Kahonnohak If I switch the Remaining era <-> amount position, how about the Available to withdraw and Re-lock section? It would be like this:
> 
> <img alt="Screenshot 2024-01-16 at 1 11 39 PM" width="895" src="https://private-user-images.githubusercontent.com/33358068/297191091-a5b0a7ce-bdb3-417f-8d59-fa471a4504b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU0NTA0MDEsIm5iZiI6MTcwNTQ1MDEwMSwicGF0aCI6Ii8zMzM1ODA2OC8yOTcxOTEwOTEtYTViMGE3Y2UtYmRiMy00MTdmLThkNTktZmE0NzFhNDUwNGI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE3VDAwMDgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhMWVkZDYxOWU1OTI2YzRlMmM5Njc2NThjNTM2M2QxM2Q5ZTc0ZmU4OTU5YjE1MGJmYjI3MWQ3MDUxYWEzMmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5sshNVU-b1Uvl5jqEi5V6AqMTFw5pwV2BWgfm3c2iwM">

My bad, of course. Sorry no change needed for this. Thank you for checking.

Kahonnohak commented:
Thank you! lets double check the details on Astar mainnet!

Kahonnohak commented:
Two problems on Asset page (another PR?)
- On this PR I see the rewards box next to the account but on the other PR I don't see it.
- Lets change the order 
Account
SBY
Staking
(project)
Assets

Kahonnohak reviewed (approved):
LGTM

ayumitk commented:
> - On this PR I see the rewards box next to the account but on the other PR I don't see it.
> - Lets change the order

@Kahonnohak I've removed the rewards box and changed the order

impelcrypto commented on src/i18n/en-US/index.ts:
```diff
@@ -742,8 +742,8 @@ export default {
     rewardsWillBeClaimed: 'Your unclaimed rewards will be claimed.',
-    basicRewards: 'Basic rewards',
-    bonusRewards: 'Bonus rewards',
+    basicRewards: 'Available to claim',
```
```suggestion
    availableToClaim: 'Available to claim',
```

impelcrypto commented on src/i18n/en-US/index.ts:
```diff
@@ -762,13 +762,21 @@ export default {
     myDapps: 'My dApps',
     unbonding: 'Unbonding',
-    totalEstimatedRewards: 'Total estimated rewards',
+    totalEstimatedRewards: 'Claim estimated Rewards',
```
```suggestion
    claimEstimatedRewards: 'Claim estimated Rewards',
```

bobo-k2 reviewed (approved):
LGTM
